include/QuadratureLfo.h

```diff
@@ -30,51 +30,68 @@
 class QuadratureLfo
 {
 public:
-	QuadratureLfo( int sampleRate );
+	QuadratureLfo( int sampleRate ) :
+		m_frequency(0),
+		m_phase(0),
+		m_offset(D_PI / 2)
+	{
+		setSampleRate(sampleRate);
+	}
+
 	~QuadratureLfo()
 	{
 	}
 
 	inline void setFrequency( double frequency )
 	{
-		if( frequency < 0 || frequency > ( m_samplerate / 2.0 ) || frequency == m_frequency )
+		if( frequency < 0 || frequency > m_samplerate / 2.0 || frequency == m_frequency )
 		{
 			return;
 		}
 		m_frequency = frequency;
 		m_increment = m_frequency * m_twoPiOverSr;
 
-		if( m_phase >= F_2PI )
+		if( m_phase >= D_2PI )
```

Comment:
I'm not sure why the clamping code is only in here. Do you have any ideas?

Reply:
It would be much better relocated after ```m_phase`` is incremented in the ```tick()``` function, I would say an unnoticed bug in my original code :( .

Reply:
Okay, I'll add a commit and add you as a co-author.